canonical-websites/build.snapcraft.io
# [Build on demand] Added 'Build now' button to repo page

## Done

Added 'Build now' button to repo page (visible only to repository owner).
List of builds is now split between latest builds (on top of the page) and previous page below help boxes.

## QA

- Check out this feature branch
- Run the site using the command ```npm start -- --env=environments/dev.env```
- View the site locally in your web browser at: [http://0.0.0.0:8000/](http://0.0.0.0:8000/)
- Go to repo page
- Builds should be split into latest/previous
- You should be able to trigger builds manually


## Issue / Card

Fixes #938

## Screenshots

<img width="1059" alt="screen shot 2017-08-31 at 13 07 19" src="https://user-images.githubusercontent.com/83575/29920346-62eb313e-8e4d-11e7-8e34-b471f34343ff.png">
